lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -2570,10 +2570,9 @@ int s1ap_handle_enb_initiated_reset_ack(
   uint32_t length = 0;
   s1ap_message message = {0};
   S1ap_ResetAcknowledgeIEs_t *s1ap_ResetAcknowledgeIEs_p = NULL;
-  S1ap_UE_associatedLogicalS1_ConnectionItemRes_t
-      sig_conn_list[MAX_NUM_PARTIAL_S1_CONN_RESET]               = {0};
-  S1ap_MME_UE_S1AP_ID_t mme_ue_id[MAX_NUM_PARTIAL_S1_CONN_RESET] = {0};
-  S1ap_ENB_UE_S1AP_ID_t enb_ue_id[MAX_NUM_PARTIAL_S1_CONN_RESET] = {0};
+  S1ap_UE_associatedLogicalS1_ConnectionItemRes_t* sig_conn_list = NULL;
+  S1ap_MME_UE_S1AP_ID_t* mme_ue_id                               = NULL;
+  S1ap_ENB_UE_S1AP_ID_t* enb_ue_id                               = NULL;
```

Comment:
@rsarwad, do you know if these are freed somewhere? After running the relevant `enb_partial_reset` s1ap test we're seeing some memory leaks after stopping MME.

Reply:
Hi @ardzoht , these memories are freed as part of fun, free_s1ap_resetacknowledge.
Yes after execution of partial reset procedure, there were 3 leaks. 2 are fixed.
1 more is found while decoding Partial Reset message. I am working on this.